portfolio/src/main/webapp/style.css

```diff
@@ -7,75 +7,78 @@
     very light grey :: #CCCCCC;
 */
 
-body, html {
+body, html  {
```

Comment:
Do these all have 2 spaces now? Is that normal css style?

Reply:
yes the normal css style is two spaces

Reply:
my mistake I looked at the style guidelines again.